modules/appeals_api/spec/fixtures/valid_200995_extra.json

```diff
@@ -31,22 +31,10 @@
             "attributes": {
               "locationAndName": "X-Ray VAMC",
               "evidenceDates": [
```

Comment:
I condensed these lines while working on this as it helped with readability. I'm happy to revert back, but I think this is much friendlier.